### ISSUE TITLE
Android OS/PlayServices/Store getting past host block

### ISSUE THREAD

**Issue by English106** [blokadaorg/blokada#480]
### Steps to reproduce
1. Add my hostfile which targets 1e100.net, google trackers [Link I use in Blokada](https://raw.githubusercontent.com/English106/ANTdroid/master/hosts)
2. Sync
3. Reboot phone and turn on blockada with VPN

### Expected behaviour

android OS, google play services, and google play store, along with various installed apps won't reach out to known 1e100.net subdomains on my hostfile

### Actual behaviour

AppSnitch app and Blokada both report in the logfiles, connections to blocked entries

### Configuration
1. Blokada version
4.1.6

2. Device model
Pixel 2 xl

3. Android version / ROM
Latest version of 9/pie

### Questions
1. How long have you been using Blokada? 
Since 3.x

2. How long time ago has the problem started to happen? 
Recent, stopped using blokada but saw there were updates which included a VPN in 4.x so I started using blokada since it's the only adblocker and vpn in-one that is open source

3.  How often does this problem happen to you?
Daily

I subscribe to the Blokada VPN and use the New York gateway.

I created a hostfile that mostly targets 1e100.net, googles +20,000 domain name cluster of trackers. 1e100.net shows on android's AppSnitch app as the most frequent connection my phone makes.

For scrutiny:
[ANTdroid.txt](https://github.com/blokadaorg/blokada/files/3505780/ANTdroid.txt)
[Link I use in Blokada](https://raw.githubusercontent.com/English106/ANTdroid/master/hosts)

Many of the connections originate from android system, google play services and play store, and a few other installed apps.

Both AppSnitch and Blokada logfiles regularly report connections to sites in my hostfile. Between my hostfile and the 3 others I have turned on there are about 100,000 entries in total which should be blocked.

Please assist.

**Comment by kar** [comment]
Are you sure it's not about subdomains? If you block a domain, it doesn't mean any subdomains from that domain are blocked.

**Comment by English106** [comment]
I'm embarassed to say it appears excel generated my text list using a special character, that looks exactly like a space.

Enabling just my hostfile blokada said 7 rules were in effect. 
I opened notepad and used replace all x with y; where x was a space and y was 5 spaces. Only 7 entries changed.
I copied the space in the other entries and used the same replace all x with y dialogue box in notepad, and my cursor was suddently sitting half an inch to the right in the x box, so I replaced it all with a space and now all entries are indentified.
